libs/ngx-dsfr/tile/src/lib/tile.component.ts

```diff
@@ -48,16 +53,34 @@ export class DsfrTileComponent implements OnChanges {
       throw EMPTY_LINK_ERROR;
     }
 
-    this.template = this.link.indexOf('http') > -1
-      ? TemplateType.EXTERNAL
-      : TemplateType.INTERNAL;
+    this.setTemplateType();
+  }
+
+  private setTemplateType(): void {
+    this.template = TemplateType.INTERNAL;
+
+    if (
+      this.link &&
+      this.link.indexOf('http') > -1 &&
+      !this.download
+    ) {
+      this.template = TemplateType.EXTERNAL
+    }
+
+    if (this.download) {
+      this.template = TemplateType.DOWNLOAD
+    }
   }
 
   private initClasses(): void {
```

Comment:
OK, turns out we never understood properly what these breakpoint classes were supposed to do , also, they changed ... 🤷🏻‍♂️
```typescript
private initClasses(): void {
    this.classes = `fr-tile fr-enlarge-link fr-tile--${this.size} fr-tile--${this.align}`;

    if (this.breakpoint) {
      const breakpointAlignment =
        this.align === 'horizontal'
          ? 'fr-tile--vertical'
          : 'fr-tile--horizontal';
      this.classes += ` ${breakpointAlignment}@${this.breakpoint}`;
    }

    if (this.download) {
      this.classes += ' fr-tile--download';
    }
  }
```